graspy/inference/latent_distribution_test.py

```diff
@@ -45,24 +54,39 @@ class LatentDistributionTest(BaseInference):
         are used for a two-sample hypothesis test on the latent positions of
         two graphs. See :class:`hyppo.ksample.KSample` for more information.
 
-    metric : str or function, (default="euclidean")
-        Distance metric to use, either a callable or a valid string.
-        The callable should behave similarly to :func:`sklearn.metrics.pairwise_distances`,
-        if a string should be one of the keys in `sklearn.metrics.pairwise.PAIRED_DISTANCES`
-        or "gaussian" which will use a Gaussian kernel on Euclidean distances
-        with an adaptively selected bandwidth.
+    metric : str or function, (default="gaussian")
+        Distance or a kernel metric to use, either a callable or a valid string.
+        If a callable, then it should behave similarly to either
+        :func:`sklearn.metrics.pairwise_distances` or to
+        :func:`sklearn.metrics.pairwise.pairwise_kernels`.
+        If a string, then it should be either one of the keys in either
+        `sklearn.metrics.pairwise.PAIRED_DISTANCES` or in
+        `sklearn.metrics.pairwise.PAIRWISE_KERNEL_FUNCTIONS`, or "gaussian",
+        which will use a gaussian kernel with an adaptively selected bandwidth.
+        It is recommended to use kernels (e.g. "gaussian") with kernel-based
+        hsic test and distances (e.g. "euclidean") with all other tests.
 
     n_components : int or None, optional (default=None)
         Number of embedding dimensions. If None, the optimal embedding
         dimensions are found by the Zhu and Godsi algorithm.
         See :func:`~graspy.embed.selectSVD` for more information.
 
     n_bootstraps : int (default=200)
-        Number of bootstrap iterations for the backend hypothesis test. See
-        :class:`hyppo.ksample.KSample` for more information.
+        Number of bootstrap iterations for the backend hypothesis test.
+        See :class:`hyppo.ksample.KSample` for more information.
 
     workers : int, optional (default=1)
         Number of workers to use. If more than 1, parallelizes the code.
+        Supply -1 to use all cores available to the Process.
+
+    size_correction: bool (default=True)
+        The size degrades in validity as the sizes of two graphs diverge from
```

Comment:
replace "size" w/ "test" i think

Reply:
hmmm... i don't feel too strongly about having this specific name, but i do not like "test_correction" too much, because it implies that it isn't correct without this flag. which is true, but only when graphs don't have the same number of vertices.

i can't remember whether i meant "size" as the number of vertices, or "size" as the thing that's less than alpha under the null. i feel like it was the former, but cep has convinced me that the number of vertices of the graph should be called "order of the graph" and not the "size of the graph", which is the number of edges, so maybe "order_correction" is more appropriate. thoughts?

Reply:
-_- didn't realize you were talking about the description... my bad

Reply:
will still change to "order_correction"

Reply:
yeah my comment was just on the description. but, i don't think "order" makes a ton of sense for n_verts to the average user. i was fine with `size_correction`. But you think "size" sounds like size of the test to a stats person?

Reply:
idk i always think of size as n_verts and not n_edges. maybe there's a third term that can make everyone happy tho

Reply:
still not loving the word "order" from a software perspective. any other terms we can think of?

Reply:
i mostly don't want to annoy graph people who are used to the convention stated in wikipedia (see email i forwarded to you). the size_correction was intuitive to me as well, personally.

Reply:
how much do you hate "correction"? i know i was initially opposed to not being clear about the effects only being present in the networks of different number of vertices, but this grew on me.

Reply:
alternatives:
- sampling
- variance_correction

Reply:
i take "sampling" back. i actually don't like it due to possible confusion with the permutations.

Reply:
lol https://www.merriam-webster.com/dictionary/incommensurable

Reply:
i'm reverting it back to size_correction, and keeping it as True by default. we can eventually just phase out uncorrected test completely, as cep suggested, but for now i think there is no harm with leaving it as an option. it would be useful at least for me to have if i need any additional simulations for my paper, and maybe someone wants to reproduce old simulations.

Reply:
agree and agree